runners/direct-java/src/test/java/org/apache/beam/runners/direct/DirectMetricsTest.java

```diff
@@ -125,6 +129,55 @@ public void testApplyAttemptedCountersQueryOneNamespace() {
             committedMetricsResult("ns1", "name1", "step2", 0L)));
   }
 
+  private boolean matchesScopeWithSingleFilter(String actualScope, String filter) {
+    Set<String> scopeFilter = new HashSet<String>();
+    scopeFilter.add(filter);
+    return metrics.matchesScope(actualScope, scopeFilter);
+  }
+
+  @SuppressWarnings("unchecked")
```

Comment:
Every test method (i.e. annotated with `@Test`) in this file has the `@SuppressWarnings("unchecked")` annotation. I just copied it - Should I remove it?